foundations/javascript_basics/fundamentals-4.md

```diff
@@ -1,3 +1,9 @@
+### Introduction
+
+One of the challenges of programming is dealing with large amounts of data. For example, if you want to store the names of all the students in your class, how would you do it? You could create a variable for each name, but that would be tedious and inefficient, not to mention hard to manage and update. For example, what if you want to change or access the names later?
```

Comment:
[Proofreading] This paragraph is off to a great start. It just needed a bit of rewording to make it clear. :) 
```suggestion
One of the challenges of programming is dealing with large amounts of data. For example, if you want to store the names of all the students in your class, how would you do it? You could create a variable for each name, but that would be tedious and inefficient. It'll also be hard to manage and update. What if you want to change or access the names later?
```

Reply:
Good changes.